pages/requests/[id].js

```diff
@@ -21,15 +23,23 @@ const Request = () => {
   const { id } = router.query
   const { request, isLoadingRequest, isRequestError } = useOneRequest(id)
   const { allSOWs, isLoadingSOWs, isSOWError } = useAllSOWs(id, request?.identifier)
-  console.log(allSOWs)
+  const { messages, isLoadingMessages, isMessagesError } = useAllMessages(id)
 
-  if (isLoadingRequest || isLoadingSOWs) return <Loading wrapperClass='item-page' />
-  if (isRequestError || isSOWError) return <h1>{`${isRequestError.name}: ${isRequestError.message}`}</h1>
+  const isLoading = isLoadingRequest || isLoadingSOWs || isLoadingMessages
+  const isError = isRequestError || isSOWError || isMessagesError
+
+  if (isLoading) return <Loading wrapperClass='item-page' />
+  if (isError) return [isRequestError, isSOWError, isMessagesError].map((err, index) => err && (
+    <>
+      <h3>{index + 1}. {err.name}:</h3>
+      <p>{err.message}</p>
+    </>
+  ))
```

Comment:
more robust error handling until we have an actual component for this. re: https://github.com/scientist-softserv/webstore-component-library/issues/25